app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -42,6 +43,12 @@ import org.mozilla.fenix.mvi.getAutoDisposeObservable
 import org.mozilla.fenix.mvi.getManagedEmitter
 import kotlin.math.roundToInt
 
+fun SessionBundleStorage.archive(sessionManager: SessionManager) {
+        save(sessionManager.createSnapshot())
+        sessionManager.removeAll()
```

Comment:
Do we also want to remove custom tab sessions? That's what removeAll() does.

Reply:
good point, also private tabs